[2022-08-11 03:34:16,146] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: craw.cr1 manual__2022-08-11T03:34:12.774368+00:00 [queued]>
[2022-08-11 03:34:16,203] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: craw.cr1 manual__2022-08-11T03:34:12.774368+00:00 [queued]>
[2022-08-11 03:34:16,205] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-11 03:34:16,205] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-08-11 03:34:16,206] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-11 03:34:16,279] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): cr1> on 2022-08-11 03:34:12.774368+00:00
[2022-08-11 03:34:16,296] {standard_task_runner.py:52} INFO - Started process 4944 to run task
[2022-08-11 03:34:16,309] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'craw', 'cr1', 'manual__2022-08-11T03:34:12.774368+00:00', '--job-id', '58', '--raw', '--subdir', 'DAGS_FOLDER/newtest.py', '--cfg-path', '/tmp/tmprdr_n06z', '--error-file', '/tmp/tmp27ocp_1b']
[2022-08-11 03:34:16,317] {standard_task_runner.py:80} INFO - Job 58: Subtask cr1
[2022-08-11 03:34:16,565] {task_command.py:371} INFO - Running <TaskInstance: craw.cr1 manual__2022-08-11T03:34:12.774368+00:00 [running]> on host d59619969b18
[2022-08-11 03:34:17,037] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=craw
AIRFLOW_CTX_TASK_ID=cr1
AIRFLOW_CTX_EXECUTION_DATE=2022-08-11T03:34:12.774368+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-11T03:34:12.774368+00:00
[2022-08-11 03:34:17,042] {logger.py:11} INFO - ====== WebDriver manager ======
[2022-08-11 03:34:17,587] {logger.py:11} INFO - Get LATEST chromedriver version for google-chrome 104.0.5112
[2022-08-11 03:34:19,389] {logger.py:11} INFO - Driver [/home/***/.wdm/drivers/chromedriver/linux64/104.0.5112/chromedriver] found in cache
[2022-08-11 03:37:42,200] {logging_mixin.py:115} INFO - 1
[2022-08-11 03:41:25,115] {logging_mixin.py:115} INFO - 2
[2022-08-11 03:44:32,084] {logging_mixin.py:115} INFO - 3
[2022-08-11 03:44:32,246] {python.py:173} INFO - Done. Returned value was: None
[2022-08-11 03:44:32,476] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=craw, task_id=cr1, execution_date=20220811T033412, start_date=20220811T033416, end_date=20220811T034432
[2022-08-11 03:44:32,633] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-11 03:44:32,797] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
